tools/c7n_gcp/c7n_gcp/resources/securitypolicy.py

```diff
@@ -0,0 +1,19 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n_gcp.provider import resources
+from c7n_gcp.query import QueryResourceManager, TypeInfo
+
+
```

Comment:
can you rename the module to armor.py to reflect underlying service, security policy is a bit generic and has potential overlap into iam, it seems like these are from the cloud armor service.

Reply:
fixed